docs/en/latest/references/apisix_consumer_v2.md

```diff
@@ -0,0 +1,46 @@
+---
+title: ApisixConsumer/v2
+keywords:
+  - APISIX ingress
+  - Apache APISIX
+  - ApisixConsumer
+description: Reference for ApisixConsumer/v2 custom Kubernetes resource.
+---
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+## Spec
+
+See the [definition](../../../../samples/deploy/crd/v1/ApisixConsumer.yaml) on GitHub.
+
+| Field            | Type    | Description                                                                                                                                    |
+|------------------|---------|------------------------------------------------------------------------------------------------------------------------------------------------|
+| authParameter          | object   | Configuration of one of the available authentication plugins.                                                                                                   |
+| authParameter.basicAuth.value   | object   | Plugin configuration for [basic-auth plugin](https://apisix.apache.org/docs/apisix/plugins/basic-auth/)                                      |
+| authParameter.basicAuth.secretRef.name   | string   | You can store plugin configuration in Kubernetes secret and reference here with the secret name                                     |
+| authParameter.keyAuth.value   | object   | Plugin configuration for [key-auth plugin](https://apisix.apache.org/docs/apisix/plugins/key-auth/)
+| authParameter.keyAuth.secretRef.name   | string   | You can store plugin configuration in Kubernetes secret and reference here with the secret name.                                    |
+| authParameter.jwtAuth.value   | object   | Plugin configuration for [jwt-auth plugin](https://apisix.apache.org/docs/apisix/plugins/jwt-auth/)
+| authParameter.jwtAuth.secretRef.name   | string   | You can store plugin configuration in Kubernetes secret and reference here with the secret name.                                    |
+| authParameter.wolfRBAC.value   | object   | Plugin configuration for [wolfRBAC-auth plugin](https://apisix.apache.org/docs/apisix/plugins/wolf-rbac/)
```

Comment:
`wolfRBAC-auth` -> `wolf-rbac`: should this and other plugins consistently use the original plugin names?

Reply:
the name of plugins in CRD and in Apisix are different. I mixed them up in the table. Okay I will stick to the name mentioned in Apisix, here.

Reply:
done